code/modules/hydroponics/hydroponics.dm

```diff
@@ -362,6 +362,13 @@
 	pestlevel = 0 // Reset
 	update_icon()
 	visible_message("<span class='warning'>The [oldPlantName] is overtaken by some [myseed.plantname]!</span>")
+	name = "hydroponics tray ([myseed.plantname])"
+	if(myseed.product)
+		var/obj/item/P = new myseed.product

```

Comment:
don't instance a new thing here just to delete it immediately after.
`initial(myseed.product.desc)` _should_ work here, if it doesn't tell me and I'll do some local testing.

Reply:
it will

Reply:
@actioninja didn't work, removing the initial() didn't help either
```
	if(myseed.product)
		desc = initial(myseed.product.desc)
	else
		desc = initial(desc)
```
```
code\modules\hydroponics\hydroponics.dm:367:error: myseed.product.desc: undefined var
code\modules\hydroponics\hydroponics.dm:367:error: initial: bad variable
```

Reply:
Ah I know what's wrong here, change the definition on the `product` var in seeds.dm to be var/obj/item/product